### DISCUSSION TITLE
After update to version 1.9.2 BTCpay Server just work local

### DISCUSSION THREAD

**Discussion by PilotoCero** [btcpayserver/btcpayserver#4958] [Technical Support] [answered]
After the Update to 1.9.2 is possible to see the store using internet, but when press "Charge" show HTTP ERROR 500. 

After try to use BTCpay Server via Internet I see the request payment locally on PAYMENTS -> Invoices and when press "Checkout" it's show the same error "HTTP ERROR 500"

Using the local server everything works Ok. Show the payment window and when open the "Checkout" works too.

The BTCpay Server was working ok until I updated.

Thanks

**Comment by PilotoCero** [comment]
when I try to open an invoice created via internet connection in this windows;

![image](https://user-images.githubusercontent.com/130673051/236564042-fc5d2ce2-f476-4ca1-8115-f9f87a8c2b86.png)

I get this result:

![image](https://user-images.githubusercontent.com/130673051/236563761-f4078d6b-47f2-455b-a614-3335f29be507.png)

But when I use the app in local server using wi-fi everything work Ok.

**Reply by PilotoCero** [reply]
Solved. Work again !

**Reply by dennisreimann** [reply]
What was the problem and how did you solve it?

**Reply by PilotoCero** [reply]
The problem was this:

**After the Update to 1.9.2 is possible to see the store using internet, but when press "Charge" show HTTP ERROR 500.

After try to use BTCpay Server via Internet I see the request payment locally on PAYMENTS -> Invoices and when press "Checkout" it's show the same error "HTTP ERROR 500"

Using the local server everything works Ok. Show the payment window and when open the "Checkout" works too.

The BTCpay Server was working ok until I updated.**

For a reason I really don't understand when I turn off the LNURL start working normal again.

![WhatsApp Image 2023-05-06 at 09 24 02](https://user-images.githubusercontent.com/130673051/236623830-efb993f6-2c2f-4fd8-b3cc-9dfda6f113d1.jpeg)

**Reply by Kukks** [reply, marked as answer]
> The problem was this:
> 
> **After the Update to 1.9.2 is possible to see the store using internet, but when press "Charge" show HTTP ERROR 500.
> 
> After try to use BTCpay Server via Internet I see the request payment locally on PAYMENTS -> Invoices and when press "Checkout" it's show the same error "HTTP ERROR 500"
> 
> Using the local server everything works Ok. Show the payment window and when open the "Checkout" works too.
> 
> The BTCpay Server was working ok until I updated.**
> 
> For a reason I really don't understand when I turn off the LNURL start working normal again.
> 
> ![WhatsApp Image 2023-05-06 at 09 24 02](https://user-images.githubusercontent.com/130673051/236623830-efb993f6-2c2f-4fd8-b3cc-9dfda6f113d1.jpeg)

this is a bug that I've just fixed for the next upcoming minor version. Thank you for reporting!
